react/Breadcrumb.js

```diff
@@ -40,51 +41,43 @@ class Breadcrumb extends Component {
   }
 
   render() {
-    const {
-      slug,
-      productQuery: { product, loading },
-    } = this.props
+    const { term, categories } = this.props
 
-    if (loading || !product || !product.categories) {
+    if (!categories) {
       return null
     }
 
-    const categoriesList = this.getCategories(product.categories)
+    const categoriesList = this.getCategories(categories)
     return (
       <div className={CSS_CLASSES.BREADCRUMB}>
         <Link className={CSS_CLASSES.LINK} page="store">
-          {this.props.intl.formatMessage({ id: 'breadcrumb.home' })}
+          <HomeIcon />
         </Link>
         {categoriesList.map(({ name, value }, i) => (
           <span key={`category-${i}`}>
-            /
+            <span className="ph2">></span>
```

Comment:
Shouldn't the `>` be and svg to match the design?

Reply:
👍